boards/feather_m0/.cargo/config

```diff
@@ -2,16 +2,13 @@
 
 [build]
 target = "thumbv6m-none-eabi"
-
-[target.thumbv6m-none-eabi]
-runner = 'arm-none-eabi-gdb'
-#runner = 'probe-run --chip ATSAMD21G18A'
-
 rustflags = [
-
   # This is needed if your flash or ram addresses are not aligned to 0x10000 in memory.x
   # See https://github.com/rust-embedded/cortex-m-quickstart/pull/95
   "-C", "link-arg=--nmagic",
-  
   "-C", "link-arg=-Tlink.x",
 ]
+
+[target.thumbv6m-none-eabi]
+runner = 'arm-none-eabi-gdb'
+#runner = 'probe-run --chip ATSAMD21E17A'
```

Comment:
The feather M0 is a `ATSAMD21G18`, so this needs to change back

Reply:
Unfortunately, that value for chip doesn't work.

```
     Running `probe-run --chip ATSAMD21G18A target/thumbv6m-none-eabi/release/examples/blinky_basic`
(HOST) INFO  flashing program (19 pages / 1.19 KiB)
Error: Error while flashing

Caused by:
    0: Failed to erase flash sector at address 0x00000000.
    1: The execution of 'erase_sector' failed with code 1. This might indicate a problem with the flash algorithm.
feather_m0 $ 
```

Reply:
But, I'm happy to leave it that way, I can always change it locally if I want to use the m0.

Reply:
Changed.

Reply:
Weird... is it not properly supported on the probe-rs side?

Reply:
That's a good guess. Perhaps the next time I want to use the m0 I'll spend some time debugging it.